### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by lucasBgabriel**
@guilhermemamprin @adrianovalente  ready to CR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+## typescript and dependencies  <br />
+```
+npm install -g typescript <br />
+npm install <br />
+bower install <br /> 
+```
+
+#Running
+```gulp serve```
```
Comment (adrianovalente) [inline README.md:25]: @lucasBgabriel shouldn't it be `gulp server`?

**Comment by guilhermemamprin** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (guilhermemamprin) [inline README.md:25]: No.

**Comment by adrianovalente** [comment]
nice @lucasBgabriel but rebase your branch before merging 👍

**Pull request merged**